src/Common/src/CoreLib/System/Reflection/MemberInfo.cs

```diff
@@ -39,7 +42,14 @@ public virtual Module Module
         public virtual IEnumerable<CustomAttributeData> CustomAttributes => GetCustomAttributesData();
         public virtual IList<CustomAttributeData> GetCustomAttributesData() { throw NotImplemented.ByDesign; }
 
+#if MONO
+        public virtual extern int MetadataToken {
```

Comment:
I'd love to get the MetadataToken property unified with .NET

Reply:
Reverted